src/ontology/mondo-edit.obo

```diff
@@ -96569,8 +96569,9 @@ synonym: "keratitis, Acanthamoeba" EXACT [NCIT:C50450]
 xref: DOID:11896 {source="EFO:0007126", source="MONDO:obsolete"}
 xref: EFO:0007126 {source="MONDO:equivalentTo"}
 xref: GARD:0009285 {source="MONDO:equivalentTo"}
-xref: ICD10:B60.1+ {source="ORDO:67043/ntbt", source="Orphanet:67043"}
-xref: ICD10:H19.2* {source="ORDO:67043/ntbt", source="Orphanet:67043"}
+xref: ICD10:B60.1 {source="ORDO:67043/ntbt", source="Orphanet:67043"}
```

Comment:
Because this code is not strictly speaking in ORDO, we should remove the ORDO source from this one.. if you want to preserve provenance you should

- indicate what kind of mapping it is (equivalent, broad, narrow, etc), i.e. `source="MONDO:equivalentTo"`
- then add a new source like `source="ORDO:67043/MONDO:derivedFrom"`. This says: "This is not exactly the code that was provided by the source, but a syntactic variation".